nuxeo-core/nuxeo-core-convert/src/test/java/org/nuxeo/ecm/core/convert/service/TestConversionService.java

```diff
@@ -54,11 +74,11 @@ public void testReplaceURLsByAbsolutePaths() throws IOException {
         Blob blob = new StringBlob("<h1>Hello World</><img src=\"" + url + "\" />");
         blob.setMimeType("text/html");
 
-        DownloadService downloadService = mock(DownloadService.class);
-        doReturn(imageBlob).when(downloadService).resolveBlobFromDownloadUrl(url);
+        Function<String, Blob> blobResolver = mock(Function.class);
+        doReturn(imageBlob).when(blobResolver).apply(url);
```

Comment:
No mock is needed (that's the point of having the function), just do

    Function<String, String> blobResolver = u -> {
        return url.equals(u) ? blob : null;
    };